src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -97,7 +97,7 @@ export class MongoDatabaseTreeItem extends AzureParentTreeItem<IMongoTreeRoot> {
 	}
 
 	async executeCommand(command: MongoCommand, context: IActionContext): Promise<string> {
-		if (command.collection) {
+		if (command.collection && !command.sendToShell) {
```

Comment:
If you want to name it `sendToShell` - I think we need to refactor this method. Something like this:
```
if (!command.sendToShell) {
	if (command.collection) {
		let db = await this.getDb();
		const collection = db.collection(command.collection);
		if (collection) {
			const collectionTreeItem = new MongoCollectionTreeItem(this, collection, command.arguments);
			const result = await collectionTreeItem.executeCommand(command.name, command.arguments);
			if (result) {
				return result;
			}
		}
	} else if (command.name === 'createCollection') {
		return withProgress(this.createCollection(stripQuotes(command.arguments.join(','))).then(() => JSON.stringify({ 'Created': 'Ok' })), 'Creating collection');
	}
}

return withProgress(this.executeCommandInShell(command, context), 'Executing command');
```

My problem with the current name is that there are cases where sendToShell could be `true`, but it's not actually sent to the shell. With my above code, that's guaranteed never to happen.